### DISCUSSION TITLE
Markdown mode with toolbar

### DISCUSSION THREAD

**Discussion by adrgerez** [tsujan/FeatherPad#613] [Ideas]
Hi,
It would be nice to have a toolbar for markdown, like the one shown in the image or even in this discussion editor, perhaps toggled from the preferences. It would be perfect for write documentation in featherpad
Thanks in advance!
![markdown_toolbar](https://user-images.githubusercontent.com/12399007/145124699-7d1ba61c-bd93-4009-b102-b2f2b9625ff5.png)

**Comment by tsujan** [comment]
Hi,

What you suggested is for *rich* text editors. For example, FeatherNotes has a compact tool-bar as well as detailed drop-down and context menus for rich texts.

FeatherPad is a plain-text/code editor. It doesn't support rich text. The rich text you see when you open a source file with it is created by its syntax highlighter and isn't saved to the file.
